utils/domain-errors.ts

```diff
@@ -0,0 +1,59 @@
+import { CosmosErrors } from "@pagopa/io-functions-commons/dist/src/utils/cosmosdb_model";
+import { errorsToReadableMessages } from "@pagopa/ts-commons/lib/reporters";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { pipe } from "fp-ts/lib/function";
+import * as t from "io-ts";
+
+export const BaseError = t.type({
+  detail: NonEmptyString,
+  title: NonEmptyString
+});
+
+export type NotFoundError = t.TypeOf<typeof NotFoundError>;
+export const NotFoundError = t.intersection([
+  BaseError,
+  t.type({
+    kind: t.literal("NotFound"),
+    objectName: NonEmptyString
+  })
+]);
+
+export const toNotFoundError = (
+  title: NonEmptyString,
+  detail: NonEmptyString,
+  objectName: NonEmptyString
+): NotFoundError => ({ detail, kind: "NotFound", objectName, title });
```

Comment:
It's not "always", because this is a method that builds a `NotFoundError`, so it's the only point in which that string will be written. 
But I agree that having an enum is a best practice. 
Done in [7908370](https://github.com/pagopa/io-functions-services/pull/232/commits/7908370d8ecab3e38357c7f9418efcbf59bd5ea4)